File: exercicios/Projeto Final/Projeto Back/Projeto/Model/Cliente.cs
namespace Projeto.Model
{
    public class Cliente
    {
        public int ID { get; set; }
        public String nome { get; set; }
        public String cpf { get; set; }
        public bool ativo { get; set; }
        //public virtual ICollection<Venda> Vendas { get; set; }
    }
}

File: Projeto Angu/src/app/cliente/cliente.ts
import { Venda } from "../venda/venda";

export class Cliente{
    id?: number;
    nome?: String;
    cpf?: String;
    ativo?: boolean;

    constructor(nome?: String, cpf?: String, id?: number){
        this.id = id;
        this.nome = nome;
        this.cpf = cpf;
        this.ativo = true;
    }
}